scripts/genDockerImageTag.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Ensure the code is run at the root directory of the project
+code_host_dir=$(dirname $(dirname ${BASH_SOURCE[0]}))
+pushd ${code_host_dir} > /dev/null # Do not print the stack to standard output otherwise it will break the plugin-enclave-gradle function getDockerContainerTag
+
+source ./containers/scripts/common.sh
```

Comment:
Not an expert of bash scripts, but do we need this? And why `./` and not `../` here, as we are in `conclave-core-sdk/scripts`?
Also, not related to this PR, do you know why we have other scripts inside the `containers/scripts` directory, i.e. why are they spread around?

Reply:
Filipe knows more the details, but those scripts are needed to build the images, and since they affect the final built image, they need to be part of the hash tag.

Reply:
> Also, not related to this PR, do you know why we have other scripts inside the containers/scripts directory, i.e. why are they spread around?

Some scripts change the contents of a docker container image so they have to contribute to hash generated for the docker container tag.